examples/hello_world/reds/HelloWorld.reds

```diff
@@ -0,0 +1 @@
+native func HelloWorld() -> Void;
```

Comment:
I left out the reds file here intentionally, because it's unnecessary. This example demonstrates that this library can be used without redscript.

Reply:
I think this example should remain the way it is, without this file